acceptance/tests/environment/environment_scenario-non_existent.rb

```diff
@@ -46,9 +46,9 @@
   },
   :puppet_agent => {
     :exit_code => 1,
-    :matches => [%r{Warning.*404.*Could not find environment '#{env}'},
+    :matches => [%r{(Warning|Error).*(404|400).*Could not find environment '#{env}'},
```

Comment:
I am not aware of any recent changes/tickets regarding http status changes - not since we worked on messages propagating to the client - many months ago.

Not sure that 400 is an appropriate response for a missing environment, 404 seems right. Could it be a difference between different kinds of missing (no environment path directory vs. can not find an environment on the path)?